browser-extension/common/src/storage.ts

```diff
@@ -37,7 +37,13 @@ export const removeRecord = async (key: string): Promise<void> => {
 };
 
 export const clearAllRecords = async (): Promise<void> => {
+  const VAR = "rq_var_isExtensionEnabled";
+  const isExtensionEnabledVariableValue = await getRecord(VAR);
```

Comment:
Why not just use `setVariable` and `getVariable` methods? It contains the prefix too?